test/plugins/common.js

```diff
@@ -235,7 +236,11 @@ function avoidTraceWriterAuth(agent) {
 }
 
 function stopAgent(agent) {
-  agent.private_().stop();
+  if (agent.isActive()) {
```

Comment:
Ideally, once the comments I expressed in `index.js` are addressed, this function be removed.